_services_projects/entering-tertiary-life-event.md

```diff
@@ -14,3 +14,41 @@ project_url: "[]()"
 quote:
 media:
 ---
+To better understand the tertiary education landscape, the Tertiary Education Commission (TEC) and the Lab completed a research discovery to identify two things: what people found challenging, and what opportunities lay in the experience of entering or re-entering the tertiary education system.
+
+The tertiary education system assumes that learning is a linear path when in fact change and interruption is natural and will occur particularly for the youngest demographic. Together with TEC we found a number of challenges people face who want to enter tertiary education. These include the quality and relevance of the information, the process they follow, and where they are in their life.   
+
+The final output of the discovery was a [research report](https://www.tec.govt.nz/assets/Publications-and-others/67c259b154/Transition-to-Tertiary-Life-Event-Final.pdf) and it recommends a number of opportunities for the education sector to explore further. 
+
+## Approach and findings
+
+During the prediscovery exercise we focused on the needs of the consumer and the information and services they access. We interviewed and workshopped alongside 55 students and subject matter experts, and reviewed a wealth of research material. We looked at opportunities where things could be done differently. 
+
+We focused on four areas specific to the needs of someone wanting to enter tertiary education. We looked at what might add value for both students and their supporters and influencers. We have an opportunity to learn from and design for what are typical changes during our lives.
+
+1. “What do I need to know and understand to help me make the best choice for now and for my future?”
+
+Students and their supporters (family, peers, teachers etc) need to plan and decide their pathway into tertiary education. Not only what suits their present life but also their future aspirations. Information relevant to the students’ life, aspirations, personalities, and how different study options effect life outcomes will aid this process.
+
+2. “What and when do I need to share information about myself, and what I want to do, with others?”
```

Comment:
instead of writing `2. `, you can write `1. ` on every item, and then the markdown engine will number them for you on display. This means you don't need to renumber. You'd need to pull the paragraphs together into a bonafide list.

Reply:
If you don't mean to make a list, you can instead have section titles with numbers in them.

Reply:
That's fab. Thanks.